charts/sps-sample/README.md

```
@@ -7,3 +7,29 @@ Sample to test a refarch application.
 | refarch-gateway | Config for refarch-gateway see https://github.com/it-at-m/helm-charts/tree/main/charts/refarch-gateway | {}    |
 | frontend        | Config for frontend see https://github.com/it-at-m/helm-charts/tree/main/charts/sps-sample/charts/frontend    | {}    |
 | backend         | Config for backend see https://github.com/it-at-m/helm-charts/tree/main/charts/sps-sample/charts/backend     | {}    |
+| postgresql      | Config for postgresql see https://github.com/bitnami/charts/tree/main/bitnami/postgresql               | {}    |
```

markdownlint-cli2 (0.17.2)

Bare URL used
null

(MD034, no-bare-urls)